openassessment/templates/openassessmentblock/oa_grade_available_responses.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n %}
+{% spaceless %}
+<div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
+    <div class="openassessment problem">
+        <div class="wrapper--grid">
+            {% if title %}
+                <h3 class="openassessment__title problem__header">{% trans title %}</h3>
```

Comment:
Also with this header being conditional, you could run in to header level issues.  If it is not available there will be a gap.  Ensure the headers progress in a logical hierarchy.